pkg/ui/static/js/graph_template.handlebar

```diff
@@ -114,8 +114,8 @@
 
                         <div class="prometheus_input_group pull-left">
                            <select name="max_source_resolution_input" title="Downsampling option" id="max_source_resolution_input{{id}}">
-                              <option value="">Auto downsampling</option>
-                              <option selected="selected" value="0s">Only raw data</option>
+                              <option value="auto">Auto downsampling</option>
+                              <option value="0s">Only raw data</option>
```

Comment:
Quick silly question, why we remove `selected` here again? It changes what is by default set, right?

Reply:
Makes sense IMHO that we keep all of that logic together in that file instead of the HTML code, thanks for pointing this out!